File: src/main/java/Util/List2String.java
package Util;

import java.util.List;

/**
 * Created by cluster on 2017/6/9.
 */
public class List2String {
    public String listToString(List list, char separator) {
        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < list.size(); i++) {
            sb.append(list.get(i)).append(separator);
        }
        return sb.toString().substring(0,sb.toString().length()-1);
    }
}

File: src/test/scala/list2String.scala
/**
  * Created by cluster on 2017/6/9.
  */
object list2String {
  def main(args: Array[String]) {
    val l1 =  List(1, 2, 3, 4, 5)
    val s1 = l1.toString
    val s2 = l1.mkString(",")
println(l1)
    println(s1)
    println(s2)
  }
}
